modules/openthread/platform/radio.c

```diff
@@ -1297,6 +1296,23 @@ otError otPlatRadioEnableCsl(otInstance *aInstance, uint32_t aCslPeriod, otShort
 	return result ? OT_ERROR_FAILED : OT_ERROR_NONE;
 }
 
+otError otPlatRadioResetCsl(otInstance *aInstance)
+{
+	struct ieee802154_config config = { 0 };
+	int result;
+
+	config.csl_period = 0;
```

Comment:
minor: This seems unnecessary if the struct is set to all zeros above anyway.

Reply:
removed